validation-test/compiler_crashers_2_fixed/0138-rdar36449760.swift

```diff
@@ -0,0 +1,8 @@
+// RUN: not %target-swift-frontend %s -typecheck
+
+class C {
```

Comment:
I don't see how this test case demonstrates a problem, since C does not conform to protocol A.

Reply:
I'm about to add 2 more cases here which might be better - `foo() -> A.B` and `foo(b: B) { _ = b.answer() }`